x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/filters/filter_popover.tsx

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import './filter_popover.scss';
+
+import React, { MouseEventHandler, useState } from 'react';
+import { EuiPopover, EuiFieldText, EuiSpacer, keys } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { debounce } from 'lodash';
```

Comment:
Instead of using lodash, you could `useDebounce` from `react-use` like @dej611 did here: https://github.com/elastic/kibana/commit/9677fcc897f43f38bbec1bb7413ad1ee376800ab

Reply:
That causes  a bunch of warnings in the console:
![image](https://user-images.githubusercontent.com/4283304/92699417-d92f2b80-f34d-11ea-83a6-62e34820150d.png)